From 5412e9969217ed272df389c608634dc41e63413d Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Fri, 10 Feb 2017 10:23:24 +0800
Subject: [PATCH] arm64: dts: rockchip: clk: rk3399: support dual pll for vop
 on drm

attention:
if the vopb is for hdmi,the vopb parent clk must be vpll
and the vopl parent clk is cpll or others plls.
if the vopl is for hdmi,the vopl parent clk must be vpll
and the vopb parent clk is cpll or other plls.

Change-Id: I9056fbdbfcdb7a71f2e7ee5d57a0db8523be66ef
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi
index 1d39c05e78b5..de6edb122696 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi
@@ -42,6 +42,7 @@
 
 #include <dt-bindings/display/drm_mipi_dsi.h>
 #include <dt-bindings/display/media-bus-format.h>
+#include "rk3399-vop-clk-set.dtsi"
 
 / {
 	compatible = "rockchip,android", "rockchip,rk3399";
-- 
2.35.3

